README.md

```diff
@@ -1,158 +1,28 @@
-# 1Password Connect Ansible Collection
+<img alt="" role="img" src="https://blog.1password.com/posts/2021/secrets-automation-launch/header.svg"/>
 
-The 1Password Connect collection contains modules that interact with your 1Password Connect deployment. The modules communicate with the 1Password Connect API to support Vault Item create/read/update/delete operations.
-
-You can learn more about [Secrets Automation and 1Password Connect](https://1password.com/secrets/) on our website.
-
-## Table of Contents
+<div align="center">
+  <h1>1Password Connect Ansible Collection</h1>
+  <p>Use the <a href="https://developer.1password.com/docs/connect">1Password Connect</a> Ansible Collection to access and manage items in your 1Password vaults.</p>
```

Comment:
[non-blocking comment]
Maybe this can be made as an actionable statement? 🤔
```suggestion
  <p>Access and manage your 1Password items in your Ansible Automation Platform through your self-hosted <a href="https://developer.1password.com/docs/connect">1Password Connect server</a>.</p>
```